docs/dev/setup/server.rst

```diff
@@ -3,103 +3,100 @@
-information about the setup for programming exercises provided:
 
-.. note::
-   Be careful that you do not commit changes to ``application-artemis.yml``.
-   To avoid this, follow the best practice when configuring your local development environment:
+       athena:
+            # If you want to use Athena, refer to the dedicated configuration section. Under Administration Guide, Setup of Extension Services.
 
-   1) Create a file named ``application-local.yml`` under ``src/main/resources/config``.
-   2) Copy the contents of ``application-artemis.yml`` into the new file.
-   3) Update configuration values in ``application-local.yml``.
+Version Control & Continuous Integration Setup
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-   By default, changes to ``application-local.yml`` will be ignored by git so you don't accidentally
-   share your credentials or other local configuration options. The run configurations contain a profile
-   ``local`` at the end to make sure the ``application-local.yml`` is considered. You can create your own
-   configuration files ``application-<name>.yml`` and then activate the profile ``<name>`` in the run
-   configuration if you need additional customizations.
+If you are setting up **programming exercises**, additional configuration is required for the version control and build system.
+Refer to one of the following guides based on your preferred setup:
 
-If you use a password, you need to adapt it in
-``gradle/liquibase.gradle``.
+- **Integrated Code Lifecycle Setup (Recommended):** :ref:`Integrated Code Lifecycle Setup <Integrated Code Lifecycle Setup>`
+- **LocalVC + Jenkins Setup:** :ref:`Jenkins and LocalVC Setup <Jenkins and LocalVC Setup>`
 
+**Note:**
+If you use a password for authentication, update it in ``gradle/liquibase.gradle``.
 
 
 Run the server via Docker
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-| Artemis provides a Docker image named ``ghcr.io/ls1intum/artemis:<TAG/VERSION>``.
-| The current develop branch is provided by the tag ``develop``.
-| The latest release is provided by the tag ``latest``.
-| Specific releases like ``5.7.1`` can be retrieved as ``ghcr.io/ls1intum/artemis:5.7.1``.
-| Branches tied to a pull request can be obtained by using the tag ``PR-<PR NUMBER>``.
+| **This method provides a fast way to start Artemis for demonstration purposes.**
+| It is **not recommended** for development, as it does not support code modifications or debugging.
+
+Artemis provides a Docker image named ``ghcr.io/ls1intum/artemis:<TAG/VERSION>``:
+
+- The **current develop branch** is available under the tag ``develop``.
+- The **latest stable release** can be retrieved using the tag ``latest``.
+- **Specific releases**, such as ``7.10.8``, can be accessed with ``ghcr.io/ls1intum/artemis:7.10.8``.
+- **Branches tied to a pull request** can be obtained using ``PR-<PR NUMBER>``.
 
+For **development**, follow the [Server Setup Guide](#server-setup) to run Artemis in an IDE.
```

Comment:
`[Server Setup Guide](#server-setup)` is this expected? Looks like a syntax error in markdown?